core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -346,6 +346,7 @@ var StoryEditorPage = function() {
       nodeOutlineEditor);
     await editor.clear();
     await richTextInstructions(editor);
+    await nodeOutlineEditor.click();
```

Comment:
Can we use [actions](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/action.js) here?

Reply:
Also, why is this required?

Reply:
We show the save and cancel button when the text editor is cliced.
I thought when we do input.sendKeys(), protractor must click on the field but my assumption was wrong and hence we need to manually click on the editor for the buttons to appear.